website/notifications/emails.py

```diff
@@ -124,6 +126,11 @@ def check_node(node, event):
     return node_subscriptions
 
 
+def check_target_user(user, event):
```

Comment:
Sounds good. Let's go with `get_user_subscriptions` (plural, since this is getting all of `user's` subscriptions, right?)